test/unit/gateways/paymill_test.rb

```diff
@@ -181,6 +181,32 @@ def test_successful_store
     assert response.test?
   end
 
+  def test_store_includes_currency
+    expected_currency = 'USD'
+
+    @gateway.expects(:raw_ssl_request).with(
+      :get,
+      store_endpoint(@credit_card, expected_currency, 100),
+      nil,
+      {}
+    ).returns(successful_store_response, successful_purchase_response)
+
+    @gateway.store(@credit_card)
+  end
+
+  def test_store_includes_amount
```

Comment:
Why have these set up as two separate tests? It looks like both of these tests are passing in both `currency` and `amount`, so I'm not seeing the difference between these tests.